content/en/os/1.15.x/api/settings/cloudformation/_index.markdown

```diff
@@ -0,0 +1,14 @@
++++
+title="cloudformation"
+type="docs"
+toc_hide=true
+description="Settings related to CloudFormation signaling (`settings.cloudformation.*`)"
++++
+
+You can setup Bottlerocket to send successful host creation or update signals to [AWS CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html).
```

Comment:
We probably want to remain consistent with the docs here: https://github.com/bottlerocket-os/bottlerocket/tree/develop/sources/cfsignal - notably, we have a few links to the upstream docs that might be helpful for a user trying to sort out how CloudFormation wants users to think about signaling.